common/membership/rpServiceResolver.go

```diff
@@ -264,6 +267,40 @@ func (r *ringpopServiceResolver) refreshNoLock() error {
 	return nil
 }
 
+func (r *ringpopServiceResolver) getReachableMembers() ([]string, error) {
+	members, err := r.rp.GetReachableMemberObjects(swim.MemberWithLabelAndValue(RoleKey, r.service))
+	if err != nil {
+		return nil, err
+	}
+
+	var hostPorts []string
+	for _, member := range members {
+		servicePort := r.port
+
+		// Each temporal service in the ring should advertise which port it has its gRPC listener
+		// on via a RingPop label. If we cannot find the label, we will assume that that the
+		// temporal service is listening on the same port that this node is listening on.
+		servicePortLabel, ok := member.Label(RolePort)
+		if ok {
+			servicePort, err = strconv.Atoi(servicePortLabel)
+			if err != nil {
+				return nil, err
+			}
+		} else {
+			r.logger.Warn("unable to find roleport label for ringpop member. using local service's port", tag.Service(r.service))
```

Comment:
maybe debug log?
the new release will create some warn log which can be misleading

Reply:
sounds good. changed